jhub_apps/tests/tests_e2e/test_integration.py

```diff
@@ -50,6 +50,7 @@ def test_panel_app_creation(playwright: Playwright, with_server_options) -> None
         create_users(page, users=[share_with_user])
         page.goto(BASE_URL)
         sign_in_and_authorize(page, username=f"admin-{app_suffix}", password="password")
+        page.goto(BASE_URL, wait_until="networkidle")
```

Comment:
This test was flaky for me unrelated to the main point of this PR.  Adding this line seemed to fix it for me.

Reply:
Update: Still seems flaky, but that line helped.  I believe the issue is unrelated to this PR.

Reply:
Update: I believe the flakiness I was seeing was due to a github issue